### PR TITLE
Chocolatey install method

### PR DESCRIPTION
**Pull request by MarkGibbons**
### Description

Install nodejs on windows servers using chocolatey

### Issues Resolved

[List any existing issues this PR resolves]

### Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by MarkGibbons** [comment]
There is no windows test in the ci/cd.  This feature needs to be tested locally.

File: recipes/nodejs_from_chocolatey.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include_recipe 'chocolatey'
+
+chocolatey_package 'nodejs-lts' do
+  version node['nodejs']['version'] if node['nodejs']['version']
+  action :upgrade
```
Comment (ramereth) [inline recipes/nodejs_from_chocolatey.rb:23]: Is the upgrade action required?

Comment (ma499) [reply recipes/nodejs_from_chocolatey.rb:23]: Without the upgrade action chocolatey will not install if an older version is already installed. In chocolatey “upgrade” means “install or upgrade if the target version (or newer) is not installed”

**Comment by ramereth** [comment]
@MarkGibbons did you intend on the windows platform to be included on any of the other suites or just the chocolately suite? I tried a few of the other suites and they aren't working.

**Comment by MarkGibbons** [comment]
Nope, windows should have been excluded from the other test suites.

File: test/cookbooks/test/recipes/chocolatey.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apt_update 'update'
```
Comment (ramereth) [inline test/cookbooks/test/recipes/chocolatey.rb:1]: I doubt we need apt related resources for things that use chocolatey.

**Review by ramereth — APPROVED** [review]

**Comment by MarkGibbons** [comment]
@ramereth thanks for hanging in there on the reviews.  I'm fitting in work on this during 10 minute breaks and early mornings mostly.  Working 4 different branches so every once in a while the responses go wrong.  Appreciate the comments.

**Comment by ramereth** [comment]
@MarkGibbons no problem! Thanks for putting up with my reviews! I'll get this released soon.

**Pull request merged**